priv/repo/migrations/20200305171000_add_slug_to_companies.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Companies.Repo.Migrations.AddSlugToCompanies do
+  use Ecto.Migration
+
+  def change do
+    alter table(:companies) do
+      add :slug, :string
+    end
+
+    create index(:companies, [:slug], unique: true)
```

Comment:
Let's change this to use the `unique_index/2`: `create unique_index(:companies, [:slug])`

Reply:
Was contemplating this as well. So 👍